flake.nix

```diff
@@ -125,7 +127,10 @@
             rustToolchain
             # cargo-package # Comment in when you want tests to run on every new shell
           ]
-          ++ pkgs.lib.optionals (pkgs.stdenv.isLinux) (with pkgs; [cargo-watch]); # Currently broken on macOS
+          ++ pkgs.lib.optionals pkgs.stdenv.isDarwin
+          [pkgs.darwin.apple_sdk.frameworks.Security]
+          ++ pkgs.lib.optionals pkgs.stdenv.isLinux
+          (with pkgs; [cargo-watch]); # Currently broken on macOS
```

Comment:
Would you happen to have a chance to test if this still holds true?
Ref: https://nav-it.slack.com/archives/C8H9NLWG0/p1686335751598759

Reply:
sure, I will have a look